python/tk_hiero_export/version_creator.py

```diff
@@ -356,6 +356,13 @@ def finishTask(self):
             version_data=vers,
         )
 
+        # Log usage metrics
+        try:
+            self.app.log_metric("Version Create", log_version=True)
+        except:
+            # ingore any errors. ex: metrics logging not supported
+            pass
+
 
```

Comment:
Ah, makes sense. I've just modified the action being logged to "Transcode & Publish" to keep it short. I don't think we need to log the version creation too. Can always come back and add if we need to.